src/core_plugins/kibana/public/home/components/tutorial/instruction_set.js

```diff
@@ -143,6 +148,39 @@ export class InstructionSet extends React.Component {
       );
     }
 
+    let launchDashboard;
+    if (this.props.overviewDashboard) {
+      launchDashboard = (
+        <div>
+          <EuiHorizontalRule />
+
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexEnd">
+
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <p>
+                  {`Once all steps are completed, you're ready to explore your data`}
+                </p>
+              </EuiText>
+            </EuiFlexItem>
+
+            <EuiFlexItem
+              grow={false}
+            >
+              <KuiLinkButton
```

Comment:
I assume the `EuiLinkButton` or the equivalent isn't available yet?

Reply:
thats correct. There is an open issue for it - https://github.com/elastic/eui/issues/24